h/static/scripts/group-forms/components/test/CreateEditGroupForm-test.js

```diff
@@ -11,6 +11,11 @@ let config;
 describe('CreateEditGroupForm', () => {
   let fakeCallAPI;
   let fakeSetLocation;
+  let fakeUseWarnOnPageUnload;
+
+  function pageUnloadWarningActive() {
+    return fakeUseWarnOnPageUnload.getCall(-1).args[0] === true;
```

Comment:
I assume `getCall(-1)` returns the last call, so perhaps using `lastCall` (which I guess it's an alias) is more clear.

```suggestion
    return fakeUseWarnOnPageUnload.lastCall.args[0] === true;
```

Reply:
> I assume getCall(-1) returns the last call

It does, like Python and numpy-style reverse indexing.